fonts/generate.py

```diff
@@ -83,21 +82,20 @@ def generate_smufl(opts: Namespace) -> bool:
     :param opts: A set of options from the argument parser sub-command.
     :return: True if successful, False otherwise.
     """
-    supported_glyphs_fname: str = opts.supported
-    supported_pth: Path = Path(supported_glyphs_fname)
+    supported_pth: Path = Path(opts.supported)
     header_out_pth: Path = Path(opts.header_out)
     header_file_pth: Path = Path(header_out_pth, "smufl.h")
 
     if not supported_pth.is_file() or not os.access(supported_pth, os.R_OK):
-        log.error("Could not find or read %s file.", supported_glyphs_fname)
+        log.error("Could not find or read %s file.", supported_pth)
         return False
 
     if not os.access(header_out_pth, os.W_OK):
         log.error("Could not write to %s.", header_out_pth)
         return False
 
     log.debug("SMuFL header will be stored in %s", header_out_pth)
-    supported_glyphs: dict = __get_supported_glyph_codes(supported_glyphs_fname)
+    supported_glyphs: dict = __get_supported_glyph_codes(supported_pth)
```

Comment:
same

Reply:
This would actually cause a problem with `.name` since the full (relative) path is needed, and `.name` would only provide the filename.